llms/mlx_lm/tuner/utils.py

```diff
@@ -7,6 +7,39 @@
 from .lora import LoRALinear
 
 
+def linear_to_lora_layers(model: nn.Module, num_lora_layers: int):
+    """
+    Convert some of the models linear layers to lora layers.
+
+    Args:
+        model (nn.Module): The neural network model.
+        num_lora_layers (int): The number of blocks to convert to lora layers
+        starting from the last layer.
+    """
+    if model.model_type in [
+        "mistral",
```

Comment:
Nit: It would be easier for future updates/modifications if we could make those model types as constant and define them at the top of utils.py.

Reply:
I think this function is going to turn into a sequence of `if elif` depending on the model types, making each one a constant might not make sense? (e.g. see the branch with `olmo`)

I admit it's not super clean though. I couldn't think of a better approach yet though.

I also want to add a config option to choose which layers (e.g. to make it easy to add MLP layers). Will do it in a follow up probably.

Reply:
Make sense, and the function already at top of  utils.py. it wouldn't be too bad to updating it.